pkg/solana/txm/pendingtx.go

```diff
@@ -66,11 +70,21 @@ type finishedTx struct {
 	state       TxState
 }
 
+type txInfo struct {
```

Comment:
removed state from pendingTx and moved it to sig level, as we now want to track state per sig